### ISSUE TITLE
Support mmap data with Python binding

### ISSUE THREAD

**Issue by V-E-O** [google/re2#224]
Hi,

I'm working on using RE2 of Python bindings(abseil branch) to search contents in large files.
The code is as below which is working with official Python RE module:
```python
with open('tmp', 'rb') as fd:
  mm = mmap.mmap(fd.fileno(), 0, access=mmap.ACCESS_READ)
  re.search(r'.+', mm)
```

Do you have plans to add similar support in RE2 Python bindings?
Thanks!

**Comment by junyer** [comment]
Thanks for the great suggestion! I have implemented this in commit d1a7120, which corresponds to version 0.0.4 on PyPI. Please confirm that it works for your use case.

**Comment by V-E-O** [comment]
Great, thanks! It's working.
